### PR TITLE
Mi open reorg

### PR DESCRIPTION
**Pull request by Rmalavally**

### CODE REVIEW

File: docs/conceptual/Getting_Started_FusionAPI.md [hunk 1]
```diff
@@ -206,4 +206,6 @@ The following graph depicts the speedup gained for a fused Convolution+Bias+Acti
 
 Speedup obtained by fusing Batchnorm (spatial mode) with Activation are presented in the graph below:
 
-![Batchnorm activation fusion](data/na.png)
```
Comment (LisaDelaney) [inline docs/conceptual/Getting_Started_FusionAPI.md]: you're in the conceptual folder, so if you want to reference something in the data folder, you need to use ".." in the path to 'go up' a folder. e.g. "[alt text](../data/conceptual/na.png)". Also note that you'll have to create the 'conceptual' folder within the data folder, as the data folder should have the same structure as the main docs folder. e.g. images that are referenced in the files in the tutorials folder should be present in a data/tutorials folder

Comment (Rmalavally) [reply docs/conceptual/Getting_Started_FusionAPI.md]: That's good to know. Do we know if this has been done before? Could you please share an example folder/template? Thanks so much.

Comment (Rmalavally) [reply docs/conceptual/Getting_Started_FusionAPI.md]: Nevermind. Randy shared an example.

**Comment by LisaDelaney** [comment]
replaced with #2783

**Pull request closed without merging**